src/QMCWaveFunctions/Fermion/SlaterDetBuilder.cpp

```diff
@@ -1150,8 +1150,7 @@ bool SlaterDetBuilder::readDetListH5(xmlNodePtr cur,
       APP_ABORT("Unknown HDF5 CI format");
     }
 
-    if (!hin.readEntry(temps[grp], ds_tag))
-      APP_ABORT("Unknown HDF5 CI format");
+    hin.read(temps[grp], ds_tag);
```

Comment:
In this case, the built-in message won't be as accurate as the explicit error message. So the old one is better but please change the APP_ABORT to runtime_error

Reply:
Yeah I thought about overloading read to take an informative message to pass to runtime_error. We could replace some `readEntry` in some other places as well if we did that. 

I reverted this changed though and throw runtime_error as requested.